aiida/README.md

```diff
@@ -0,0 +1,38 @@
+# {{ name }}
+{% if description %}
+{{ description }}
+{% endif %}
+
+{% if archive_url %}
+This project imports an [AiiDA archive]({{ archive_url }}) upon starting an environment, which may take some time.
+{% endif %}
+
+
+### Introduction
+
+This project comes with an AiiDA environment already pre-configured for your user.
+
+ 1. Start a new environment (e.g. 1 CPU, 2GB memory)
+ 1. (optional) inspect the `aiida_config.yaml` in case you would like to make any changes
```

Comment:
since this is automatically generated, any changes the user makes to the file itself will be overwritten...

Reply:
right, that was one of the items to improve in the readme ;-)

I've removed the point